src/Algebra/Graph.hs

```diff
@@ -445,10 +445,10 @@ hasEdge = H.hasEdge
 -- @
 -- vertexCount 'empty'      == 0
 -- vertexCount ('vertex' x) == 1
--- vertexCount            == 'length' . 'vertexList'
+-- vertexCount            == 'Data.Set.size' . 'vertexSet'
```

Comment:
This can be simplified to just `Set.size`.

Reply:
I just have a doubt: Do you want: `'Set.size' . 'vertexSet'` or just `'Set.size'` ?

Reply:
I mean just drop the redundant `Data.` :)

Reply:
Oh, wait. I'm sorry, but I think I might have been wrong here. Haddock might actually need `Data.` to link the proper documentation entry. Could you check this, please?

Reply:
Yes, I agree, and `length . vertexList` is simpler to understand